llms/speculative_decoding/speculative.py

```diff
@@ -59,12 +56,12 @@ def main(args):
     parser.add_argument(
         "--model-name",
         help="Name of the model.",
-        default="t5-small",
+        default="t5-base",
     )
     parser.add_argument(
         "--draft-model-name",
         help="Name of the draft model.",
-        default="t5-small",
+        default="t5-base",
```

Comment:
Shouldn't we use a large `t5-large` for the main model and a `t5-small` for the base? Otherwise doesn't seem like there is much point to specualitive decoding. I think I set bad defaults before but maybe we should fix now.

Reply:
yes! seems I managed to push some debugging settings.